modules/dicom_archive/README.md

```diff
@@ -0,0 +1,42 @@
+# Dicom Archive
+
+## Purpose
+
+The DICOM Archive module provides a front end to view the metadata
+about DICOMs which have been inserted into the LORIS pipeline.
+
+## Intended Users
+
+The DICOM Archive is primarily used by DCC staff who are investigating
+the status of imaging pipeline scripts.
+
+## Scope
+
+The DICOM Archive shows DICOM tar metadata. It does not provide a
+way to visualize scans or view metadata of other data formats. It
+provides a frontend to the `tarchive_*` tables in LORIS, but does
+not directly inspect files on the filesystem.
+
+## Permissions
+
+The permission `dicom_archive_view_allsites` is required to access
+the DICOM Archive module.
```

Comment:
There are no permissions for own site in this module? Note for later, could be useful, no?

Reply:
yeah, but currently the hasAccess just checks for "dicom_archive_view_allsites"

Reply:
Should we create a Redmine task for this?

Reply:
Sure.